OmniKit/OmnipodCommon/MessageBlocks/DetailedStatus.swift

```diff
@@ -160,21 +160,21 @@ extension DetailedStatus: RawRepresentable {
 }
 
 extension TimeInterval {
-    var stringValue: String {
-        let totalSeconds = self
-        let minutes = Int(totalSeconds / 60) % 60
-        let hours = Int(totalSeconds / 3600) - (Int(self / 3600)/24 * 24)
-        let days = Int((totalSeconds / 3600) / 24)
-        var pluralFormOfDays = "days"
-        if days == 1 {
-            pluralFormOfDays = "day"
+    var timeIntervalStr: String {
```

Comment:
nit: The name `timeIntervalStr` is redundant with the type it's defined on (both have "time interval" in it), and the formatting is non-localized. Might be ok since it's just used for debug display. Maybe `formattedDebugDescription` would be better.  Also this approach (adding a very custom, non localizable formatter to a global, widely used type) is a little heavy.  Now `5.timeIntervalStr` is a thing.

Reply:
I don’t understand the 5.timeIntervalStr comment as “5” is an Int, not a TimeInterval. In any case, I didn’t create this approach, I just changed the var name of this already existing approach from “stringValue” to “timeIntervalStr” as I found the later more helpful when reading the source code for the comms structs and I thought it appropriate to use a new name for the new format (which I found more helpful when reading parsed comms logs). So my personal preference would still be to use timeIntervalStr, but I’d be willing if needed to change everything over to use the longer `formattedDebugDescription` name to get this PR landed.